data/json/itemgroups/Locations_MapExtras/locations.json

```
@@ -347,7 +347,7 @@
       [ "radio", 10 ],
       [ "eyedrops", 10 ],
       [ "rag", 30 ],
-      [ "paper", 30 ],
+      { "item": "paper", "prob": 30, "count": 50 },
```

```suggestion
      { "item": "fork", "count": [ 25, 75 ], "prob": 30 },
```
Count I made is arbitrary, but randomizing the amount you find would be appreciated & realistic.